utilities/rev.py

```diff
@@ -0,0 +1,29 @@
+import rev
+import wpilib
+
+
+class SparkMaxAbsoluteEncoderWrapper:
+    def __init__(self, motor: rev.CANSparkMax) -> None:
+        self.real_encoder = motor.getAbsoluteEncoder(
+            rev.SparkMaxAbsoluteEncoder.Type.kDutyCycle
+        )
+        self.sim_position = 0
+        self.sim_velocity = 0
```

Comment:
```suggestion
        self.sim_position = 0.0
        self.sim_velocity = 0.0
```
This should be the fix to the mypy stuff